lab_1/main.py

```diff
@@ -14,9 +10,9 @@ def calculate_frequences(text):
     frequency_dict = {}
     if type(text) is str:
         text1 = text.lower()
-        signs = """?!%~<>/\.,[]{}()@"'#*&0123456789"""
+        signs = """^;:$`?№!%~<>/\.,[]{}()@"'#*&0123456789"""
         for c in signs:
-            text1 = text1.replase(c, " ")
+            text1 = text1.replaсe(c, " ")
```

Comment:
@KateAntsiferova it's a cyrillic 'c', not latin 'c' in replace.  Check locally before commiting